src/containers/Permission.jsx

```diff
@@ -1,32 +1,64 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 
-import { APPS_DOCTYPE } from 'doctypes'
+import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
 import { withRouter } from 'react-router-dom'
 import Typography from 'cozy-ui/transpiled/react/Typography'
-import { Q, useQuery, isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
+import CozyClient, {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
 import { useI18n } from 'cozy-ui/transpiled/react'
 import Page from 'components/Page'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
-import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import { displayPermissions } from './helpers/permissionsHelper'
 
 const Permission = ({ match }) => {
   const { t } = useI18n()
   const appName = match.params.app
   const permissionName = match.params.permission
+  const THIRTY_SECONDS = 30 * 1000
 
-  const queryResult = useQuery(
+  const [verbs, setVerbs] = useState([])
+
+  const queryResultApps = useQuery(
     Q(APPS_DOCTYPE).getById('io.cozy.apps/' + appName),
     {
-      as: 'io.cozy.apps/' + appName
+      as: 'io.cozy.apps/' + appName,
+      fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
     }
   )
 
+  const queryResultKonnectors = useQuery(
+    Q(KONNECTORS_DOCTYPE).getById('io.cozy.konnectors/' + appName),
+    {
+      as: 'io.cozy.konnectors/' + appName,
+      fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
+    }
+  )
+
+  useEffect(() => {
+    let result
+    if (queryResultApps.data.length > 0) {
+      result = queryResultApps
+    } else if (queryResultKonnectors.data.length > 0) {
+      result = queryResultKonnectors
+    } else {
+      return
+    }
+    setVerbs(result.data[0].attributes.permissions[permissionName].verbs)
```

Comment:
You shouldn't access data through `attributes`, as mentioned yesterday, this is a bug from cozy-client ;)
https://github.com/cozy/cozy-client/issues/1118
Also, you might want to be more protective, i.e. avoid app crash if there is a missing field, we now have the `?` operator for this : https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Optional_chaining

So as a result, you can simply do:
```
const verbs = result.data?.permissions?.[permissionName]?.verbs`
if (verbs) {
  setVerbs(verbs)
}
```